rskj-core/src/main/java/co/rsk/core/bc/BlockExecutor.java

```diff
@@ -219,11 +219,11 @@ private boolean validateLogsBloom(BlockHeader header, BlockResult result) {
 
     @VisibleForTesting
     public BlockResult execute(Block block, BlockHeader parent, boolean discardInvalidTxs) {
```

Comment:
Minor thing, but this `@VisibleForTesting` could be package-private.

Reply:
done